docs/code_ref/index.rst

```diff
@@ -7,18 +7,19 @@ Code Reference
 .. toctree::
    :maxdepth: 2
 
-   sunpy
    coordinates/index
    data
    database
    image
    io
+   known_issues
```

Comment:
I feel like known issues and stability should go by themselves after the list of sub-modules?

Reply:
Good point, I sorted it since that's how I roll.